tests/Client/Helpers/LiveTestFixture.cs

```diff
@@ -28,11 +28,22 @@ public override async ValueTask InitializeAsync()
         {
             Environment.CurrentDirectory = Settings.SettingsDirectory;
             transportOptions.Command = "npx";
+#if DEBUG
```

Comment:
I'm not sure how useful the DEBUG flag is in a test project.   Projects default to debug and we haven't chanted that default, so I would assume that all of our tests are run in debug mode unless someone manually changes to Release before running tests.

But the test's build configuration is orthogonal to the build configuration used when building the package or wanting to be able to attach a debugger at server start.

I think this fits better as a test setting.  i.e.  There's no reason to worry about what gets compiled into the test dll, so I'd avoid compilation conditionals and just rely on the TestSettings poco and json file

Reply:
great point, I moved it to the LiveTestSettings.cs, so one can turn this on-off using `.testsettings.json`